.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
I'm note sure this project will EVER be big enough to require writing queries in CodeQL to check the codebase for rough coding practices (all the code can be reviewed manually faster than writing tests for it because it's so small)? I think we should hold off on adding all these tools to such a tiny application, but thanks very much for your interest in contributing! :)